nucleus/admin/rest/rest-client/src/main/java/org/glassfish/admin/rest/client/utils/MarshallingUtils.java

```diff
@@ -77,12 +75,12 @@
         json = json.trim();
         if (json.startsWith("{")) {
             properties = new ArrayList<Map<String, String>>();
-            properties.add(processJsonMap(json));
+            properties.add(Util.processJsonMap(json));
```

Comment:
You may want to statically import `Util.processJsonMap` here, as the fact that `processJsonMap` is now in a separate utility class doesn't really add much info to the reader who tries to understand this code.

Reply:
I hate statically imported methods! Methods should be in the class (or a parent one), not just grabbed from some random place at the top of the file.

Reply:
You could ask yourself what the difference is between writing `java.lang.String` and `Util.processJsonMap` vs writing `String` and `processJsonMap`. In both cases the location they come from does not help with the readability.

The above example makes it even more clear. It was `processJsonMap` before. Then it became `Util.processJsonMap`. Does the latter make it really more clear to read?

Reply:
Yes, especially since the method is no longer coming from the within the class, which it was previously. As there already was an identical method in another class in the same package it now uses that instead.